dist_threshold: 0.35
hover_duration: 3.5
start_settle_duration: 7.0
#mission_settle_duration: 6.0
gripper_latency: 0.0


grasp_target: [0.257, -3.463, 0.989]
start_position: [-2.0, 0.0, 2.0]

# y: + is toawrds us
# z: + is down
#offset: [0.00, -0.025, -0.14] #vision slow and close
#offset: [0.00, -0.025, -0.22] #vision
#offset: [0.0, 0.05, -0.21] #mocap
#offset: [0.0, 0.01, -0.19] #mocap cardboard box
#offset: [0.0, 0.02, -0.18] #mocap pepsi 0.5 m/s
offset: [0.0, 0.0, -0.21] #mocap pepsi 1.25 m/s
#offset: [0.0, 0.03, -0.18] #mocap medkit
#offset: [0.0, 0.03, -0.18] #mocap fast pepsi


open_distance: 0.2
#grasp_velocity: [0.2, 0.0, -0.029]
#grasp_velocity: [0.0, 0.0, 0.0]
grasp_speed_horz: 2
grasp_speed_vert: -0.02
ground_effect_stop_time: 5.5

#end_offset: [1.6, 0.0, 1.0]
end_offset_horz : 1.7
end_offset_vert: 1.0 
end_velocity: [0.0, 0.0, 0.0]
end_time: 5.0

desired_yaw: -1.57
trajectory_plane: "xz"

grasp_lockout_distance: 1.0

initial_lengths: [190, 190, 208, 208]

# Time into the future when the grasp should occur
grasp_time: 4
use_hardcoded_grasp_time: True

average_polynomial_velocity: 0.4
minimum_polynomial_time: 3
